### PR TITLE
Add data structure file

### LINKED ISSUE #16: Data file
This file should have the data that will be displayed to the user. This file will be used as a template until the BE team has the `endpoint` needed to consume the data.

### PR DESCRIPTION
**Pull request by Atos20**
### What does this PR do?
Create dummy data that will be used to display the survey 

### Where should the reviewer start?
`src/data.tsx`
### How should this be manually tested?

- navigate to the folder and open the file to see the data 

### Any background context you want to provide?

- This data will be used to display survey
- 
### What are the relevant tickets?
closes #16 
### Screenshots (if appropriate)
Screen Shot 2021-01-06 at 1 18 23 PM

### CODE REVIEW

File: src/data.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
+export const data_local = {
+
+  "data": {
+
+    "annual_salary":{
```
Comment (timkeresey) [inline src/data.tsx:5]: I hadn't considered whether interfaces can be nested when defined, so this may be something we need to look into with typescript.

File: src/data.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    },
+    "survey":{
+      "type": "survey",
+      "question": "",
+      "description": "While every person’s situation is different we aim to provide you with a report that would give you a better perspective on where you are on your dream home quest. We based our data on your net monthly income (that’s after taxes), monthly debt, income, downpayment and more",
+      "information": "One of the main questions to answers is how much can I afford?. Affordability is defined as the cost of something.",
+      "note": "Here are the generally recommended guidelines: * Your mortgage payment should be 28% or less. * Your debt-to-income ratio (DTI) should be 36% or less. * Your housing expenses should be 29% or less. This is for things like insurance, taxes, maintenance, and repairs. * You should have three months of housing payments and expenses saved up.",
+      "source": "http://www.trulia.com"
+    }
+  }
```
Comment (timkeresey) [inline src/data.tsx:93]: I think this covers all the data needed.

**Review by timkeresey — APPROVED** [review]

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
+.eslintcache
```
Comment (Arique1104) [inline .gitignore:2]: Humble questions from a Back End Engineer
Why was this added to .gitignore?

File: src/_variables.css [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  --numbers_color: #DB4C40;
+  --report_green: #89BD9E;
+  --dark_purple: #3C153B;
+  --claret: #8B1E3F;
+  --title: 'Gravitas One', cursive;
+  --description: 'Xanh Mono', monospace;
+  --banner: 'East Sea Dokdo', cursive;
+  --note: 'Quicksand', sans-serif;
+  --error: 'Gravitas One', cursive;
+}
```
Comment (Arique1104) [inline src/_variables.css:19]: Is this our beautiful color palette?!?!?!

File: src/data.tsx [hunk 5]
```diff
@@ -0,0 +1,94 @@
+export const data_local = {
+
+  "data": {
+
+    "annual_salary":{
+      "type": "Annual Salary",
```
Comment (Arique1104) [inline src/data.tsx:6]: FYI, python has a method called `type` 
We changed `"type":` to `"classification":` in our model.

File: src/data.tsx (hunk 2 above)
Comment (Arique1104) [reply src/data.tsx:93]: Heads up:

Backend discovered some CRUD functionality that we think would be useful.

Say, in the future, you want to update the source link and description for once of these items.  When we add this functionality, you'll be able to post, update, delete, on top of get.

It's currently outside of MVP, but it's coming.

File: src/data.tsx [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+      "source": ""
+    },
+    "external_services": {
+      "type": "",
+      "property_tax": "number",
+      "description": "",
+      "information": "",
+      "note":"",
+      "affordability": "mortgage amount + average mortgage from zipcode"
+    },
```
Comment (Arique1104) [inline src/data.tsx:84]: We can get rid of this section, it goes into a different model we're calling the Report Model.

**Review by Arique1104 — CHANGES_REQUESTED** [review]
This looks good, there are some changes I'm requesting just so that there is more cohesion with what Backend will be providing Frontend.

**Review by Arique1104 — APPROVED** [review]
